src/pages/en/guides/integrations-guide.md

```diff
@@ -15,6 +15,8 @@ title: Using Integrations
 
 In this example, we will add the `@astrojs/react` integration to add React support to your Astro project. The process for adding any other framwork (Preact, Vue, Svelte or Solid.js) is almost identical and can be followed using the same steps outlined below.
 
+> Feeling adventurous? Astro recent launched an **experimental** `astro add` command to automate this process! If you'd like to try it, just run `astro add react`. That's it!
```

Comment:
Nit: should we say `npx astro add` in case the astro CLI isn't configured as a global? I know this is a windows issue in particular.

```suggestion
> Feeling adventurous? Astro recent launched an **experimental** `npx astro add` command to automate this process! If you'd like to try it, just run `astro add react`. That's it!
```